src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs

```diff
@@ -2,28 +2,25 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using NuGet.Common;
+using NuGet.VisualStudio.Telemetry;
 
 namespace NuGet.VisualStudio.Common.Telemetry.PowerShell
 {
-    public class InstanceCloseEvent : TelemetryEvent
+    public static class InstanceCloseEvent
```

Comment:
Since this is not at event model class, I would rename it to something that reflects its purpose:

```suggestion
    public static class InstanceCloseTelemetryEmitter
```